src/pybritive/britive_cli.py

```diff
@@ -542,7 +542,7 @@ def construct_kube_config(self, from_cache_command=False):
 
         profiles = []
         for p in self.available_profiles:
-            if p['app_type'].lower() == 'kubernetes':
+            if p['app_type'].lower() == 'kubernetes' and p['env_properties']:
```

Comment:
Will this be present but `None` or should we do a `p.get()`?

Reply:
Will always be present, since we're using `env.get('profileEnvironmentProperties', {})` to set `'env_properties'`